File: problemas/iniciante/2717/2717.cpp
#include <iostream>

using namespace std;

int main()
{
    int N, A, B;

    cin >> N;
    cin >> A >> B;

    if (A + B <= N)
    {
        cout << "Farei hoje!" << endl;
    }
    else
    {
        cout << "Deixa para amanha!" << endl;
    }

    return 0;
}

File: problemas/iniciante/2717/2717.kt
import java.util.*

fun main(args: Array<String>) {
  val sc = Scanner(System.`in`)

  val n = sc.nextInt()
  val a = sc.nextInt()
  val b = sc.nextInt()

  if(a + b <= n){
    println("Farei hoje!")
  }else{
    println("Deixa para amanha!")
  }
}
